dcolor.py

```diff
@@ -56,18 +56,20 @@ def plot(self, f, xdim=10, ydim=8, plt_dpi=100):
         plt.gca().invert_yaxis() #make CCW orientation positive
         plt.gca().get_xaxis().set_visible(False)
         plt.gca().get_yaxis().set_visible(False)
+        return plt
+
+    def showPlot(self, f, xdim=10, ydim=8, plt_dpi=100):
+        plt = self.getPlot(f, xdim=10, ydim=8, plt_dpi=100)
```

Comment:
should we be passing in the arguments here instead of assigning them (here and elsewhere)? i.e. `showPlot` accepts an argument `xdim` that defaults to `10`, so we don't have to assign it again in the call to `getPlot`